pages/[user].js

```diff
@@ -69,6 +69,10 @@ const UserPage = () => {
         <UserFeedStatus {...user}/>
       </p>
 
+      {offset > 0 && (
+        <PaginationLinks offset={offset} username={username}/>
+      )}
+
```

Comment:
Hmm... I'm thinking about moving this piece of logic inside the component. How about instead of that `if`, we add another (optional) prop, `hideOnFirst`, and then rely on it inside the component, returning `false` if that is provided.